.github/workflows/sandbox/1.4-M3-eap.diff

```diff
@@ -140,7 +140,7 @@ index 331c1f05..4e9e89aa 100644
            )
          }
 diff --git a/compiler-plugin/src/main/kotlin/arrow/meta/phases/resolve/synthetics/SyntheticScopeProvider.kt b/compiler-plugin/src/main/kotlin/arrow/meta/phases/resolve/synthetics/SyntheticScopeProvider.kt
-index b71d5ea9..7335517c 100644
+index b71d5ea..7335517 100644
```

Comment:
Are these changes expected? Not sure what it does. /cc @rachelcarmena

Reply:
They aren't necessary. I removed the next version checks until we merge the upgrade to 1.4-M3 (WIP). Thanks for letting me know :raised_hands: